forem/forem
# Score and flag count fix in users profile in admin section

## What type of PR is this? (check all applicable)


## Description

Reference: https://forem-team.slack.com/archives/CAA200SH5/p1688134053094819

## Related Tickets & Documents



- Related Issue #19492

## QA Instructions, Screenshots, Recordings

1. Sign in as admin
2. Mark a user as flagged
3. Mark an article by that user as flagged
4. Comment on the article created by that same user and mark that as flagged.
5. Now visit `admin/member_manager/users` for that specific user and it `Flags` tab
6. Notice that the vomit count and score count is shown for `user` flagged only and it does not take `article` and `comment` into consideration.

Issue and output explanation: https://www.loom.com/share/17f81505921d48d992c67e8459aed57d?sid=34062a5a-e4b6-4bc0-b60e-1f95ba15a4ae

### UI accessibility checklist
No UI change

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
